airlock/templates/file_browser/directory.html

```diff
@@ -1,16 +1,61 @@
-{% #card title=path_item.name %}
-  {% #list_group %}
-    {% if not path_item.children %}
-      {% list_group_empty icon=True title="This directory is empty" %}
-    {% else %}
-      {% for entry in path_item.children %}
-        {% #list_group_item href=entry.url %}
-          {{ entry.name}}
-          {% if entry.is_directory %}
-            {% icon_folder_outline class="h-6 w-6 text-slate-600 inline" %}
-          {% endif %}
-        {% /list_group_item %}
-      {% endfor %}
-    {% endif %}
-  {% /list_group %}
+{% load django_vite %}
+{% load airlock %}
+{% load static %}
+
+<style>
+  #customTable thead {
+    position: sticky;
+    top: 0;
+    text-align: left;
+    background-color: rgba(248,250,252);
+  }
+
+  #customTable td.name a.directory {
+    background-image: url(/static/folder.png);
+    background-repeat: no-repeat;
+    background-size: 1.4rem;
+    background-position: left -0.2rem top 0;
+    padding-left: 1.1rem;
+  }
+
+</style>
```

Comment:
I think @tomodwyer recently put all the custom CSS into a separate file - can we do that here?

Reply:
He did?

Hmm, that might cause some problems.

Reply:
https://github.com/opensafely-core/airlock/pull/314/commits/1a56f1a70d063d635d5bfb50b598511de1aa621b

Reply:
Hmm, I am not sure we want all datatables to have sticky header.

I am a fan of keeping style local to their use.  The current datatables implementation makes this a bit more awkward, as you have to use #customTable

Reply:
Also the the a.directory styles are very specific to *this* table, not others.

Reply:
I think ultimately we'll want this to move out of the template & into a CSS file, but it doesn't have to be in this pr. If all the tables are `#customTable`, could we put an id on the parent card & refer to it like `#directoryTable #customTable thead` etc?